cocos/2d/CCNode.cpp

```diff
@@ -1741,25 +1741,25 @@ const Mat4& Node::getNodeToParentTransform() const
                 _transform.m[13] += _transform.m[1] * -_anchorPointInPoints.x + _transform.m[5] * -_anchorPointInPoints.y;
             }
         }
+        
+        if (_additionalTransformDirty)
+        {
+            // This is needed to support both Node::setNodeToParentTransform() and Node::setAdditionalTransform()
+            // at the same time. The scenario is this:
+            // at some point setNodeToParentTransform() is called.
+            // and later setAdditionalTransform() is called every time. And since _transform
+            // is being overwritten everyframe, _additionalTransform[1] is used to have a copy
+            // of the last "_transform without _additionalTransform"
+            if (_transformDirty)
+                _additionalTransform[1] = _transform;
+            
+            if (_transformUpdated)
+                _transform = _additionalTransform[1] * _additionalTransform[0];
+        }
+        
+        _transformDirty = false;
```

Comment:
why `_additionalTransformDirty = false;` was removed? 
isn't it needed anymore here? thanks.